C/c4CAPI.cc

```diff
@@ -33,6 +33,12 @@ using namespace fleece;
 using namespace litecore;
 
 
+#define checkCollectionValid(c, err, ret) if(_usuallyFalse(!c4coll_isValid(c))) { \
```

Comment:
Any reason to use #define instead of static inline?

Reply:
Want it to return from the method it is placed in.  You cannot do that with a static inline.

Reply:
Maybe rename it to make that clearer, since it's unusual behavior? Like, `returnIfCollectionInvalid`?